lib/parallel_cucumber_worker/cli/main.js

```diff
@@ -1,5 +1,6 @@
 var Main = function() {
   var Path = require('path');
+  var Command = require('commander').Command;
```

Comment:
@simondean 
Yes, the command library has been introduced in cucumber-js for the CLI arguments handling, so that needed to be changed here as well.